app/presenters/tasks/base_task.rb

```diff
@@ -10,6 +10,10 @@ def initialize(crime_application:)
       @crime_application = crime_application
     end
 
+    def name
```

Comment:
Moved name here so that we can use the task subclass rather than a sparate task presenter.